config/routes.rb

```diff
@@ -0,0 +1,31 @@
+Rails.application.routes.draw do
+  # get 'home/index'
+  
```

Comment:
- [ ] Kindly delete commented code to follow[best practise](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)